ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -204,12 +204,16 @@ export const WorkspaceAbout = fp.flow(withUserProfile(), withUrlParams(), withCd
     return <div style={styles.mainPage}>
       <FlexColumn style={{margin: '1rem', width: '98%'}}>
         <ResearchPurpose data-test-id='researchPurpose'/>
-        {profile.authorities.includes(Authority.FEATUREDWORKSPACEADMIN) &&
-        <div style={{display: 'flex', justifyContent: 'flex-end'}}>
-            <Button disabled={publishing} type='secondary'
-                    onClick={() => this.publishUnpublishWorkspace(false)}>Unpublish</Button>
-            <Button onClick={() => this.publishUnpublishWorkspace(true)}
-                    disabled={publishing} style={{marginLeft: '0.5rem'}}>Publish</Button>
+        {hasAuthorityForAction(profile, AuthGuardedAction.PUBLISH_WORKSPACE) &&
+          <div style={{display: 'flex', justifyContent: 'flex-end'}}>
```

Comment:
We should move this to the workspace admin page eventually - I filed https://precisionmedicineinitiative.atlassian.net/browse/RW-5786